src/tools/rbd_mirror/image_replayer/BootstrapRequest.cc

```diff
@@ -375,29 +376,173 @@ void BootstrapRequest<I>::handle_open_local_image(int r) {
   update_client_image();
 }
 
-template <typename I>
+template<typename I>
 void BootstrapRequest<I>::create_local_image() {
   dout(20) << dendl;
 
-  m_local_image_id = "";
   update_progress("CREATE_LOCAL_IMAGE");
```

Comment:
The global marker set and remove is kind of an internal mechanism to handle interrupted image creation/cloning. Therefore, progress is updated with "CREATE" tag within here.